### ISSUE TITLE
Kaldi JNI

### ISSUE THREAD

**Issue by jcsilva** [iamadamhair/kaldroid#1]
Hi,

have you already written a JNI library for Kaldi? I saw you have some pocketsphinx jni jar in your project tree, but I didn't find anything similar for Kaldi.

Thank you!

**Comment by iamadamhair** [comment]
No, I haven't. We were working on this for a project last year and decided to use PocketSphinx. We may get around to Kaldi later, but it seems that PocketSphinx may work for now.
